docs/extended_api/synchronization_primitives/barrier/wait_parity.md

```diff
@@ -0,0 +1,45 @@
+---
+grand_parent: Extended API
+parent: Barriers
+---
+
+# `cuda::barrier::wait_parity` and `cuda::barrier::try_wait_parity`
+
+Defined in header `<cuda/std/barrier>`:
+
+```cuda
+__host__ __device__ void cuda::std::barrier::wait_parity(bool phase);
+__host__ __device__ bool cuda::std::barrier::try_wait_parity(bool phase);
+```
+
+`barrier::wait_parity` stalls execution while the barrier is not at the specified phase.
+`barrier::try_wait_parity` queries the the state of the barrier against the specified phase.
```

Comment:
See the previous comment. It's really weird that the description here doesn't mention parity at all.